### PR TITLE
Testimonials page

### LINKED ISSUE #4667: Add "Testimonials" page to knative.dev website
We want to capture the content from the blog post [Highlighting the Value of Knative for the C-Suite](https://knative.dev/blog/articles/highlighting-value-knative-c-suite/)  and make it a permanent page under an “About” section on the website.

Here is the design that was provided by the contractor Lionsway we worked with.

Here is the design 
[Testimonials Page.pdf](https://github.com/knative/docs/files/7936315/Testimonials.Page.pdf)

### PR DESCRIPTION
**Pull request by gabo1208**
Fixes #4667

### CODE REVIEW

**Comment by csantanapr** [comment]
it looks very good @gabo1208 
We just need to adjust the location.
We discussed in DUX meetings where to place "testimonials". rename "Case Studies" with "About" 
See for more details https://github.com/knative/docs/issues/4779

File: config/nav.yml [hunk 1]
```diff
@@ -239,6 +239,8 @@ nav:
       - Client: reference/client/README.md
       - Concepts:
         - Duck types: reference/concepts/duck-typing.md
+    - About:
```
Comment (csantanapr) [inline config/nav.yml:242]: We don't want a new menu "About"

I think this is what we are looking for
```
    - Community:
      - Welcome to the community: community/README.md
      - Contribute to Knative: community/contributing.md
      - About the community: community/about.md
    - About:
      - Testimonials: about/testimonials.md
      - Case studies:
        - about/case-studies/index.md
        - deepc: about/case-studies/deepc.md
        - Outfit7: about/case-studies/outfit7.md
        - Puppet: about/case-studies/puppet.md
    - Blog: /blog/
```

Comment (gabo1208) [reply config/nav.yml:242]: One thing that is missing here is the index on the case studies, for now I'll redirect it to the deepc as it is right now

Comment (csantanapr) [reply config/nav.yml]: yeah redirect to deepc one

**Comment by csantanapr** [comment]
again thank you @gabo1208 
/unhold
/lgtm

**Pull request merged**